source/reference/mongosync.txt

```diff
@@ -27,7 +27,15 @@ line options.
 .. note::
 
    - Both the source cluster and destination cluster must use MongoDB
-     6.0 or later.
+     6.0 or later. 
+     
+     You can migrate data on clusters (source) with versions of MongoDB
+     lower than 6.0 to an Atlas cluster. Migration from clusters with
```

Comment:
```suggestion
     lower than 6.0 to an Atlas cluster (destination). Migration from clusters with
```
If we mention `(source)`, we can also mention `(destination)` so it's clear which is which.